docs/importing.md

```diff
@@ -0,0 +1,73 @@
+# Importing existing Data into Chronicle
+
+## Importing when running on Docker compose
+
+## Importing Data into Chronicle
+
+Chronicle provides a CLI command `import` to load data from a JSON-LD file that
+contains an array of Chronicle Operations. These operations can be used to
+create agents, namespaces, and other resources in the Chronicle database.
+
+### Local development environment
+
+To import data into Chronicle, follow these steps:
+
+- Create a JSON-LD file with an array of Chronicle Operations.
+- Save the file in the Chronicle root directory with a unique name, for example,
+  `import.json`.
+- Run the following command:
+
+  ```bash
+  chronicle --bin chronicle -- import <namespace-id> <namespace-uuid> <json-file-url>
+  ```
+
+  Where:
+
+  `<namespace-id>`: the name of the namespace to which the data will be imported.
+  `<namespace-uuid>`: the UUID of the namespace to which the data will be imported.
+  `<json-file-url>`: the URL of the JSON-LD file containing the Chronicle Operations.
+
+- If the data is successfully imported, the Chronicle Operations will be added
+  to the Chronicle database.
+
+### Docker compose environment
+
+To import data into Chronicle, follow these steps:
+
+- Create a JSON-LD file with an array of Chronicle Operations.
+- Add a bind mount to your docker compose file mounting your import file in the
+  chronicle container, if you're using the chronicle-examples or chronicle
+  bootstrap configuration this will in the `docker/chronicle.yaml`
+  and is the `chronicle-sawtooth-api` container.
```

Comment:
```suggestion
  chronicle container, if you're using the chronicle-examples or
  chronicle-bootstrap configuration then this will be for the
  `chronicle-sawtooth-api` container, defined in `docker/chronicle.yaml`.
```

Reply:
Thanks, I've updated the text